tests/conftest.py

```diff
@@ -785,229 +787,203 @@ def fixture_stats_api(project_dir: Path) -> StatsAPI:
-        flow_cell_path=bcl2fastq_flow_cell_dir
-    )
-    working_flow_cell: FlowCellDirectoryData = FlowCellDirectoryData(flow_cell_path=working_dir)
-    shutil.copy(
-        existing_flow_cell.run_parameters_path.as_posix(),
-        working_flow_cell.run_parameters_path.as_posix(),
-    )
+    working_dir: Path = Path(tmp_flow_cells_demux_all_directory, bcl2fastq_flow_cell_dir.name)
     return working_dir
 
 
-@pytest.fixture(name="flow_cell_working_directory_dragen")
-def fixture_flow_cell_working_directory_dragen(
-    dragen_flow_cell_dir: Path, flow_cell_runs_working_directory_dragen: Path
+@pytest.fixture(name="tmp_flow_cell_demux_all_directory_bclconvert")
+def fixture_flow_cell_demux_all_directory_bclconvert(
+    bcl_convert_flow_cell_dir: Path, tmp_flow_cells_demux_all_directory: Path
 ) -> Path:
     """Return the path to a working directory that will be deleted after test is run.
-
+    Used to test functions in demultiplex flow cell.
     This is a path to a flow cell directory with the run parameters present.
     """
-    working_dir: Path = Path(flow_cell_runs_working_directory_dragen, dragen_flow_cell_dir.name)
-    working_dir.mkdir(parents=True)
-    existing_flow_cell: FlowCellDirectoryData = FlowCellDirectoryData(
-        flow_cell_path=dragen_flow_cell_dir
-    )
-    working_flow_cell: FlowCellDirectoryData = FlowCellDirectoryData(flow_cell_path=working_dir)
-    shutil.copy(
-        existing_flow_cell.run_parameters_path.as_posix(),
-        working_flow_cell.run_parameters_path.as_posix(),
-    )
+    working_dir: Path = Path(tmp_flow_cells_demux_all_directory, bcl_convert_flow_cell_dir.name)
     return working_dir
 
 
-@pytest.fixture(name="flow_cell_working_directory_no_run_parameters")
-def fixture_flow_cell_working_directory_no_run_parameters(
-    bcl2fastq_flow_cell_dir: Path, flow_cell_runs_working_directory: Path
+@pytest.fixture(name="tmp_flow_cell_name_no_run_parameters")
+def fixture_tmp_flow_cell_name_no_run_parameters() -> str:
+    """This is the name of a flow cell directory with the run parameters missing."""
+    return "180522_A00689_0200_BHLCKNCCXY"
+
+
+@pytest.fixture(name="tmp_flow_cell_name_ready_for_demultiplexing_bcl_convert")
+def fixture_tmp_flow_cell_name_ready_for_demultiplexing_bcl_convert() -> str:
+    """This is the name of a flow cell directory with the run parameters missing.
+    Contains a sample sheet that is bcl convert compliant
+    """
+    return "211101_A00187_0615_AHLG5GDRZZ"
+
+
+@pytest.fixture(name="tmp_flow_cell_name_no_sample_sheet")
+def fixture_tmp_flow_cell_name_no_sample_sheet() -> str:
+    """This is the name of a flow cell directory with the run parameters missing."""
+    return "170407_A00689_0209_BHHKVCALXX"
+
+
+@pytest.fixture(name="tmp_flow_cell_name_ready_for_demultiplexing_bcl2fastq")
+def fixture_tmp_flow_cell_name_ready_for_demultiplexing_bcl2fastq() -> str:
+    """This is the name of a flow cell directory with the run parameters missing."""
```

Comment:
```suggestion
    """This is the name of a flow cell directory with the run parameters missing., but otherwise complete and ready for demuliplexing."""
```